### PR TITLE
(DOCSP-22492): add asymmetric sync example - JS

### PR DESCRIPTION
**Pull request by mongomoe**
### Jira

- https://jira.mongodb.org/browse/DOCSP-22492

### Staged Changes (Requires MongoDB Corp SSO)

- [Define a Realm Object Model # Define an Asymmetric Object](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/asymmetric-sync-js/sdk/node/examples/define-a-realm-object-model/#define-an-asymmetric-object)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### Note:
- I still need to add a link to https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/asym_sync/sync/data-access-patterns/flexible-sync/#optimize-sync-with-asymmetric-sync--optional-

### CODE REVIEW

File: source/sdk/node/examples/define-a-realm-object-model.txt [hunk 1]
```diff
@@ -82,6 +82,34 @@ value or a list of values.
 To specify that a field contains a list of a primitive value type, append ``[]``
 to the type name.
 
+.. _node-define-an-asymmetric-object:
+
+Define an Asymmetric Object
+---------------------------
+
+If you are using Flexible Sync and need to sync a collection unidirectionally
```
Comment (dacharyc) [inline source/sdk/node/examples/define-a-realm-object-model.txt:90]: My only nit is that `unidirectionally` is a long and potentially difficult-to-parse word. Could we say instead: 

> sync a collection one-way from your device to your Atlas database

Or else work in the "one way" phrasing somewhere in the example below, just to help for folks who may have trouble with "unidirectionally"?

**Review by dacharyc — APPROVED** [review]
LGTM. Just one nit and a Q about the code example, but neither need block merging. Although obviously with the understanding you'll add the missing link before merging.

File: source/sdk/node/examples/define-a-realm-object-model.txt (hunk 1 above)
Comment (dacharyc) [reply source/sdk/node/examples/define-a-realm-object-model.txt:90]: It's a long word, plus it's an adverb form of a long word, so I think readers with a lower English reading level (which includes non-native speakers) will have problems with it. I'm fine with leaving it here if it's the word specified by stakeholders, but then I'd add "one way" down below in the example for additional context.

Comment (mongomoe) [reply source/sdk/node/examples/define-a-realm-object-model.txt:90]: added 'one-way' to the code snippet comment below.

**Review by kneth — COMMENTED** [review]
LGTM

**Pull request merged**